public/components/trace_analytics/components/traces/traces_content.tsx

```diff
@@ -252,6 +252,7 @@ export function TracesContent(props: TracesProps) {
               />
             </EuiFlexItem>
           </EuiFlexGroup>
+          {props.filters.length > 0 && <EuiSpacer size="s" />}
```

Comment:
This spacer can ideally to the place where we render filter badges. That is already checking this condition: `props.filters.length > 0`

Reply:
Moved into the filter.